src/Installer/PluginInstaller.php

```diff
@@ -169,6 +181,26 @@ public static function determinePlugins($packages, $pluginsDir = 'plugins', $ven
         return $plugins;
     }
 
+    /**
+     * Turns relative paths in full paths.
+     *
+     * @param string $path Path
+     * @param string $vendorDir The path to the vendor dir
+     * @return string
+     */
+    protected static function fullpath($path, $vendorDir)
+    {
+        if (preg_match('{^(?:/|[a-z]:|[a-z0-9.]+://)}i', $path)) {
+            return rtrim($path, '/');
+        }
+
+        if (substr($path, 0, 2) === './') {
+            $path = substr($path, 2);
+        }
+
+        return rtrim(dirname($vendorDir) . DIRECTORY_SEPARATOR . $path);
```

Comment:
Would you be able to use [realpath](http://php.net/manual/en/function.realpath.php) here at all?